themes/src/main/resources-community/theme/base/account/messages/messages_de.properties

```diff
@@ -94,29 +97,43 @@ revoke=Berechtigung widerrufen
 
 configureAuthenticators=Authenticatoren konfigurieren
 mobile=Mobile
-totpStep1=Installieren Sie <a href="https://freeotp.github.io/" target="_blank">FreeOTP</a> oder <a href="http://code.google.com/p/google-authenticator/" target="_blank">Google Authenticator</a> auf Ihrem Smartphone.
-totpStep2=\u00D6ffnen Sie die Applikation und scannen Sie den Barcode oder geben Sie den Code ein.
-totpStep3=Geben Sie den von der Applikation generierten One-time Code ein und klicken Sie auf Speichern.
+totpStep1=Installieren Sie eine der folgenden Applikationen auf Ihrem Smartphone:
+totpStep2=\u00D6ffnen Sie die Applikation und scannen Sie den Barcode.
+totpStep3=Geben Sie den von der Applikation generierten One-Time-Code ein und klicken Sie auf Speichern.
+
+totpManualStep2=\u00D6ffnen Sie die Applikation und geben Sie den folgenden Schl\u00FCssel ein.
+totpManualStep3=Verwenden Sie die folgenden Konfigurations-Werte, falls die Applikation erlaubt sie zu konfigurieren:
+totpUnableToScan=Sie K\u00F6nnen den Barcode nicht scannen?
+totpScanBarcode=Barcode scannen?
+
+totp.totp=Zeitbasiert (time-based)
```

Comment:
zeitbasiert

Reply:
Apart from the capitalization, would you suggest that I omit the "(time-based)" part? 

I was uncertain whether to include it or not. If we only display the German word, users might get confused, especially if their authenticator application is not or not properly translated.

Reply:
You shouldn't omit the "(time-based)" part.